daemon/mgr/volume.go

```diff
@@ -83,6 +83,9 @@ func (vm *VolumeManager) Create(ctx context.Context, name, driver string, option
 
 	v, err := vm.core.CreateVolume(id)
 	if err != nil {
+		if e, ok := err.(*volerr.CoreError); ok && e.IsVolumeExisted() {
```

Comment:
Could we have a way to combine `pkg/errtypes` and `storage/volume/error` into one?
Or extract the common part? @rudyfly